src/services/helpers/service.js

```diff
@@ -28,9 +28,9 @@ module.exports = function setup(app) {
 		// POST
 		async create(data, params) {
 			const FORCE_SEND_EMAIL = app.get('FORCE_SEND_EMAIL');
-			const NOTIFICATION_PLATFORM = app.get('NOTIFICATION_PLATFORM');
+			const notificationPlatform = app.get('NOTIFICATION_PLATFORM');
 
-			if (!NOTIFICATION_PLATFORM || process.env.NOTIFICATION_PLATFORM) {
+			if (!notificationPlatform) {
 				throw new Unavailable('Required Env NOTIFICATION_PLATFORM is not defined');
```

Comment:
I would use a 405 Method Not Allowed, as the problem is not temporary, and the client should not retry the operation.
(Server understood the message, but the function is disabled and can not be used).